### PR TITLE
create a reference for basic command 'expand'

### LINKED ISSUE #257: creating a reference for basic command "expand"

### PR DESCRIPTION
**Pull request by GaryTheGary**
# Advanced HW 7 submission

Closes #257

Name: Wenjie Fang
Uniqname: garyfang

### CODE REVIEW

**Comment by mterwill** [comment]
Looks good to me overall. Like we talked about in OH, one common task I could see needing to do is convert tabs to spaces and write that back into the same file, for which the trivial

```
$ expand my_file.txt > my_file.txt
```

would fail, as `my_file.txt` is truncated before `expand` gets the opportunity to read it. Would be cool to add a short section in your documentation with a workaround!

**Comment by GaryTheGary** [comment]
Added example and solution to possible issue.

File: _commands/basics/expand.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+`expand` a command that convert tabs to spaces.
+
+~~~ bash
+$ expand [OPTION] [FILE]
+~~~
+
+<!--more-->
+
+### Description
+ * Convert tabs in each FILE to spaces, writing to standard output. With no FILE, or when FILE is -, read standard input.
```
Comment (mterwill) [inline _commands/basics/expand.md]: Could you surround FILE and - in \`backticks\` so they get rendered as `code`?

File: _commands/basics/expand.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Tag
```
Comment (mterwill) [inline _commands/basics/expand.md]: I think "Flags" might be more appropriate here than "Tag"?

File: _commands/basics/expand.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ * `-i, --initial`, do not convert tabs after non blanks.
+ * `-t, --tabs=NUMBER`, have tabs NUMBER characters apart, not 8.
+ * `-t, --tabs=LIST`, use comma separated list of explicit tab positions.
+ * `--help`, display this help and exit
+ * `--version`, output version information and exit
+
+
+### Useful Options / Examples(Description)
```
Comment (mterwill) [inline _commands/basics/expand.md]: Maybe just change this to "Example"?

**Comment by GaryTheGary** [comment]
I just fixed couple style inside the "expand" page, let me know if anything else needs to update. Thank you!

**Comment by mterwill** [comment]
Awesome; thanks. Merging + giving you credit!

**Pull request merged**